### PR TITLE
More obvious warning in _commands

### PR DESCRIPTION
**Pull request by SamStudio8**

### CODE REVIEW

**Comment by zuzak** [comment]
The wording should be tweaked.
The bot is in the control channel, so "we" is misleading; "this is" would perhaps be more apt.

**Comment by zuzak** [comment]
Alternatively, we could just go for "you are not authorized to perform this command", and hide the whole control channel thing from end-users.

**Comment by SamStudio8** [comment]
:ship:

**Comment by zuzak** [comment]
-1. There's bad spacing, and the quote is two sentences.

`I'm sorry, samstudio8I'm afraid I can't let you do that.`

Moreover, please rebase your changes into a single commit.

**Comment by zuzak** [comment]
Also the build would fail because you're using double quotes instead of single ones.

File: modules/_commands.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ module.exports = {
 							}
 							if ( cmd.privileged ) {
 								if ( bot.config.irc.control !== msg.to ) {
-									bot.say( msg.to, msg.nick + ': Unable.' );
+									bot.say( msg.to, "I'm sorry, " + msg.nick + "I'm afraid I can't let you do that." );
```
Comment (zuzak) [inline modules/_commands.js]: Codacy Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/douglas/civilservant/file/2351755039/issues/source?bid=3060616&fileBranchId=3217890#l25)

**Comment by SamStudio8** [comment]
rekt

**Pull request closed without merging**